src/bepasty/utils/upload.py

```diff
@@ -5,6 +5,14 @@
 
 from flask import current_app
 
+try:
+    import magic
```

Comment:
how about using `... as magic_module` here and `magic` instead of `mag`?

Reply:
fixed